setup.py

```diff
@@ -49,7 +49,8 @@
     zip_safe=False,
     platforms='any',
     install_requires=[
-        'Invenio>=1.9999.4',
+        'Invenio>=2.1',
```

Comment:
:+1: for merging in the current state and :+1: for changing the dependencies to `invenio-base` and friends as soon as possible!